### DISCUSSION TITLE
Glycol Chiller & v 4.3

### DISCUSSION THREAD

**Discussion by IDMaynard** [vitotai/BrewPiLess#405] [Q&A]
My question is two part

1)
I am trying to enable the glycol chiller portion of the code.

I am comparing the config.h file from v4.2.2 and v4.3 and I see a couple of differences that I am not sure how to handle

Here ae the differences I found

![code differences](https://github.com/vitotai/BrewPiLess/assets/106493963/6fe92cbd-93b9-45b0-bb0e-59875fcca86e)

2)
I am not a coder so I am unsure how to enable the glycol chiller portion of the code.  

Do I just add True to the end of the EnableGlycolSupport and remove the // ? or are there other areas of the code that I need to modify?

I see there is a section in the uncompiled code that is specific for the ESP8266-gylcol, but the second line is commented saying it is not working yet.

Any assistance would be greatly appreciated.

**Comment by vitotai** [comment]
People complains about mysterious fridge reading without fridge sensors.
I am tired of answering questions like this.
SensorFallBack as well as configurable minimum cooling/heating time are two features to support Glycol. 
SensorFallBack is the cause that causes people complain, while configurable minimum cooling/heating time is good to have.

I could've made glycol support a setting, but I couldn't figure out a structural way without change EEPROM map.
